src/main/webapp/app/exercises/quiz/manage/quiz-management.route.ts

```diff
@@ -13,7 +13,7 @@ export const quizManagementRoute: Routes = [
         redirectTo: ':courseId/exercises',
     },
     {
-        path: ':courseId/quiz-exercises/:exerciseId',
+        path: ':courseId/quiz-exercises/new',
         component: QuizExerciseDetailComponent,
         data: {
             authorities: [Authority.TA, Authority.EDITOR, Authority.INSTRUCTOR, Authority.ADMIN],
```

Comment:
The TA is not supposed to access the quiz exercise creation view, so we should remove the respective authority